commands/utils/git.go

```diff
@@ -146,3 +166,16 @@ func (gm *GitManager) IsClean() (bool, error) {
 
 	return status.IsClean(), nil
 }
+
+func createBasicAuth(token string) *http.BasicAuth {
+	return &http.BasicAuth{
+		Username: "username", // The username can be anything except an empty string
+		Password: token,
+	}
+}
+
+// getFullBranchName returns the full branch name ( for example: refs/heads/master)
+// The input branchName can be short name (master) or full name (refs/heads/master)
```

Comment:
...be short name (master) or full name... --> ...be a short name (master) or a full name..